com.unity.render-pipelines.core/ShaderLibrary/DataExtraction.hlsl

```diff
@@ -0,0 +1,79 @@
+﻿#ifndef UNITY_DATA_EXTRACTION_INCLUDED
+#define UNITY_DATA_EXTRACTION_INCLUDED
+
+#include "Packages/com.unity.render-pipelines.core/ShaderLibrary/Packing.hlsl"
+
+// These correspond to UnityEngine.Camera.RenderRequestMode enum values
+#define RENDER_OBJECT_ID               1
+#define RENDER_DEPTH                   2
+#define RENDER_WORLD_NORMALS_FACE_RGB  3
+#define RENDER_WORLD_POSITION_RGB      4
+#define RENDER_ENTITY_ID               5
+#define RENDER_BASE_COLOR_RGBA         6
+#define RENDER_SPECULAR_RGB            7
+#define RENDER_METALLIC_R              8
+#define RENDER_EMISSION_RGB            9
+#define RENDER_WORLD_NORMALS_PIXEL_RGB 10
+#define RENDER_SMOOTHNESS_R            11
+#define RENDER_OCCLUSION_R             12
```

Comment:
We should call this RENDER_AMBIENT_OCCLUSION  as occlusion is undefined (in HDRP we have specular occlusion for example)